dropwizard/service/build.gradle.kts

```diff
@@ -168,20 +168,19 @@ tasks.register<Jar>("testJar") {
 val shadowJar =
   tasks.named<ShadowJar>("shadowJar") {
     append("META-INF/hk2-locator/default")
-    finalizedBy("startScripts")
+    finalizedBy("startShadowScripts")
   }
 
 val startScripts =
-  tasks.named<CreateStartScripts>("startScripts") {
-    classpath = files(provider { shadowJar.get().archiveFileName })
-    applicationName = "polaris-service"
-  }
+  tasks.named<CreateStartScripts>("startScripts") { applicationName = "polaris-service" }
+
+tasks.named<CreateStartScripts>("startShadowScripts") { applicationName = "polaris-service" }
 
 tasks.register<Sync>("prepareDockerDist") {
   into(project.layout.buildDirectory.dir("docker-dist"))
   from(startScripts) { into("bin") }
-  from(shadowJar) { into("lib") }
-  doFirst { delete(project.layout.buildDirectory.dir("regtest-dist")) }
```

Comment:
I am not sure why this `doFirst` section was introduced. I do not see any "regtest-dist" directory in the codebase.